vue-nagesen/src/store/index.js

```diff
@@ -20,9 +21,29 @@ export default new Vuex.Store({
     setUser(state, user){
       state.user.name = user.name;
       state.user.account = 1000;
+
+      db.collection('users').doc(firebase.auth().currentUser.uid).set({
+        'name': state.user.name,
+        'account': state.user.account
+      }, { merge: true })
     },
     setLogin(state){
       state.user.isLogin = true;
+      //const doc = db.collection('users').doc(firebase.auth().currentUser.uid).get()
+      //const user = doc.data()
+      //state.user.name = user.name
+      let userData = {}
+      const docRef = db.collection('users').doc(firebase.auth().currentUser.uid)
+      docRef.get().then((doc) => {
+        if (doc.exists) {
+          userData = doc.data()
```

Comment:
doc.data().nameという参照ができるので、オブジェクトに一度代入する必要はないかなと思います！
代入するにしても、35行目でletでオブジェクトを定義するのではなく、この行で定義すればconst定義にできます。
```suggestion
          const userData = doc.data()
```